ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentBaselines.tsx

```diff
@@ -1,14 +1,96 @@
+import { Divider, Flex, FlexItem, Stack, StackItem, Switch, Tooltip } from '@patternfly/react-core';
 import { HelpIcon } from '@patternfly/react-icons';
 
+import EntityNameSearchInput from '../common/EntityNameSearchInput';
+import AdvancedFlowsFilter, {
+    defaultAdvancedFlowsFilters,
+} from '../common/AdvancedFlowsFilter/AdvancedFlowsFilter';
+import { AdvancedFlowsFilterType } from '../common/AdvancedFlowsFilter/types';
+import { Flow } from '../types';
+import { getAllUniqPorts } from '../utils/flowUtils';
+
+const baselines: Flow[] = [
+    {
+        id: 'External Entities-Ingress-Many-TCP',
+        type: 'External',
+        entity: 'External Entities',
+        namespace: '',
+        direction: 'Ingress',
+        port: 'Many',
+        protocol: 'TCP',
+        isAnomalous: false,
+        children: [
+            {
+                id: 'External Entities-Ingress-443-TCP',
+                type: 'External',
+                entity: 'External Entities',
+                namespace: '',
+                direction: 'Ingress',
+                port: '443',
+                protocol: 'TCP',
+                isAnomalous: false,
+            },
+            {
+                id: 'External Entities-Ingress-9443-TCP',
+                type: 'External',
+                entity: 'External Entities',
+                namespace: '',
+                direction: 'Ingress',
+                port: '9443',
+                protocol: 'TCP',
+                isAnomalous: false,
+            },
+        ],
+    },
+    {
+        id: 'Deployment 1-naples-Ingress-Many-TCP',
+        type: 'Deployment',
+        entity: 'Deployment 1',
+        namespace: 'naples',
+        direction: 'Ingress',
+        port: '9000',
+        protocol: 'TCP',
+        isAnomalous: false,
+        children: [],
+    },
+    {
+        id: 'Deployment 2-naples-Ingress-Many-UDP',
+        type: 'Deployment',
+        entity: 'Deployment 2',
+        namespace: 'naples',
+        direction: 'Ingress',
+        port: '8080',
+        protocol: 'UDP',
+        isAnomalous: false,
+        children: [],
+    },
+    {
+        id: 'Deployment 3-naples-Egress-7777-UDP',
+        type: 'Deployment',
+        entity: 'Deployment 3',
+        namespace: 'naples',
+        direction: 'Egress',
+        port: '7777',
+        protocol: 'UDP',
+        isAnomalous: false,
+        children: [],
+    },
+];
+
 function DeploymentBaselines() {
     const [isAlertingOnViolations, setIsAlertingOnViolations] = React.useState<boolean>(false);
+    const [entityNameFilter, setEntityNameFilter] = React.useState<string>('');
+    const [advancedFilters, setAdvancedFilters] = React.useState<AdvancedFlowsFilterType>(
+        defaultAdvancedFlowsFilters
+    );
 
     const handleAlertingOnViolations = (checked: boolean) => {
         setIsAlertingOnViolations(checked);
     };
 
+    const allUniqPorts = getAllUniqPorts(baselines);
```

Comment:
nit: Even though Lodash abbreviates `uniquie` as `uniq`, it's confusing for not a lot of saving. (Two characters. You could say it's going to be auto-completed in most editors most of the time, but that begs the question "why save the two characters then?")

Reply:
True. I'm ok with the complete name vs. the shortened name. We can do that